tests/systemxhr.js

```diff
@@ -40,7 +45,38 @@
             log('XHR error. ' + newXHR.statusText);
           }
         };
+      }
 
+      function testXMLHttpRequestJSON() {
+        var newXHR = createXMLHttpRequest();
+
+        newXHR.onreadystatechange = evt => console.info(evt);
+        newXHR.open('get',
+                    'http://antonioma.github.io/WebAPI_pf/tests/test_list.json',
```

Comment:
We should probably get something that's always on other origin (like www.google.com for example)